CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ This changelog keeps track of all changes to the Packs SDK. We follow convention
 ### Added
 
 - `coda execute` now allows for executing metadata formulas and sync updates.
+- Add `systemConnectionAuthentication` to `ExternalPackVersionMetadata`.
```

Comment:
For internal-only stuff we needn't update the changelog, that's meant for public consumption.